src/core_plugins/kibana/public/management/sections/indices/_create.js

```diff
@@ -22,7 +22,7 @@ uiModules.get('apps/management')
 
   // this and child scopes will write pattern vars here
   const index = $scope.index = {
-    name: 'logstash-*',
+    name: config.get('indexPatternPlaceholder', 'logstash-*'),
```

Comment:
It doesn't look like you need the default value here. It'll use the `value` from the `indexPatternPlaceholder` definition in `settings/defaults.js`.